.github/workflows/ci.yml

```diff
@@ -45,3 +46,10 @@ jobs:
       with:
         name: surefire-${{ matrix.jdk-distribution }}-${{ matrix.os }}
         path: '**/surefire-reports/*.txt'
+    - name: Run Tests on Java 24
+      run: mvn -ntp -U -B -fae clean install "-Djvm=${{env.JAVA_HOME_24_X64}}/bin/java"
+    - uses: actions/upload-artifact@v4
+      if: failure()
+      with:
+        name: surefire-${{ matrix.jdk-distribution }}-${{ matrix.os }}-24
+        path: '**/surefire-reports/*.txt'
```

Comment:
We need to remember to update this once 24 is finalized, and again when the next LTS is out (to remove this extra test run).

Reply:
Yeah, I guess the version name in there is probably a bit much.